x-pack/plugins/siem/public/components/timeline/properties/index.tsx

```diff
@@ -299,14 +303,14 @@ export class Properties extends React.PureComponent<Props, State> {
 
           {title != null && title.length
             ? usersViewing.map(user => (
-                <EuiFlexItem key={user}>
+                <HiddenFlexItem key={user}>
```

Comment:
Optional: I'd add a comment about why we are hiding this just in case it's a while before we get back to this.